File: src/main/java/com/wgzhao/addax/admin/common/TaskStatus.java
package com.wgzhao.addax.admin.common;

/**
 * 采集任务状态
 *
 * @author wgzhao
 */
public class TaskStatus {
    // 采集任务状态
    public static final String COLLECTING = "R"; // 采集中
    public static final String COLLECTED = "Y"; // 采集完成
    public static final String COLLECT_FAIL = "F"; // 采集失败
    public static final String WAITING_COLLECT = "W"; // 等待采集
    public static final String EXCLUDE_COLLECT = "X"; // 不采集

    private TaskStatus() {}
}

File: src/main/kotlin/com/wgzhao/addax/admin/common/TaskStatus.kt
package com.wgzhao.addax.admin.common

/**
 * 采集任务状态
 *
 * @author wgzhao
 */
object TaskStatus {
    // 采集任务状态
    const val COLLECTING: String = "R" // 采集中
    const val COLLECTED: String = "Y" // 采集完成
    const val COLLECT_FAIL: String = "F" // 采集失败
    const val WAITING_COLLECT: String = "W" // 等待采集
    const val EXCLUDE_COLLECT: String = "X" // 不采集
}
